rust-runtime/aws-smithy-http-server/Cargo.toml

```diff
@@ -20,11 +20,22 @@ aws-smithy-http = { path = "../aws-smithy-http" }
 aws-smithy-types = { path = "../aws-smithy-types" }
 aws-smithy-json = { path = "../aws-smithy-json" }
 axum = { version = "0.3", features = [ "http1", "http2", "headers", "mime", "tower-log" ], default-features = false }
+async-trait = "0.1"
 bytes = "1.1"
+futures-util = { version = "0.3", default-features = false }
 http = "0.2"
 http-body = "0.4"
 hyper = { version = "0.14", features = ["server", "http1", "http2", "tcp"] }
 mime = "0.3"
+pin-project-lite = "0.2.7"
+regex = "1.0"
+serde_urlencoded = "0.7"
+thiserror = "1"
+tokio = { version = "1.0", features = ["full"] }
+tower = { version = "0.4" }
+
+# TODO This can be removed, it's in the service crate.
+derive_builder = "0.10"
```

Comment:
Is this TODO already applicable?

Reply:
Yes, it was here to have the example application in `main.rs` of the `aws-smithy-http-server` build.